e2e/npm_packages/packages/wrapper-lib/BUILD.bazel

```diff
@@ -0,0 +1,61 @@
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+load("@aspect_rules_js//js:defs.bzl", "js_library")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("@jest//:defs.bzl", "jest_test")
+
+npm_link_all_packages(name = "node_modules")
+
+# Basic js with 1 npm dep and one js_library() dep on a subdir
+js_library(
+    name = "wrapper-lib_jslib",
+    srcs = ["index.js"],
+    deps = [
+        "//packages/wrapper-lib/subdir",
+        "//:node_modules/@aspect-test/b",
+        "//:node_modules/@e2e/lib",
+
+        # TODO: this is transitive and should not be required?
```

Comment:
I think this is a bug which I've seen elsewhere in a non-test scenario. Without this there is a runtime "cant find module" error:
```
    Cannot find module '@aspect-test/a' from '../../node_modules/.aspect_rules_js/@e2e+lib@0.0.0/node_modules/@e2e/lib/index.js'

    Require stack:
      /.../sandbox/darwin-sandbox/403/execroot/__main__/bazel-out/darwin_arm64-fastbuild/bin/packages/wrapper-lib/test_cjs.sh.runfiles/__main__/node_modules/.aspect_rules_js/@e2e+lib@0.0.0/node_modules/@e2e/lib/index.js
      /.../sandbox/darwin-sandbox/403/execroot/__main__/bazel-out/darwin_arm64-fastbuild/bin/packages/wrapper-lib/test_cjs.sh.runfiles/__main__/tools/jest-env.js

       6 |
       7 | global.globalJestEnvInit = function() {
    >  8 |     jest.useFakeTimers();
         |                 ^
       9 |     global.foo = e2eLibId()
      10 | }

      at Resolver._throwModNotFoundError (../../../jest/node_modules/.aspect_rules_js/jest-resolve@29.3.1/node_modules/jest-resolve/build/resolver.js:425:11)
      at Object.<anonymous> (../../node_modules/.aspect_rules_js/@e2e+lib@0.0.0/node_modules/@e2e/lib/index.js:7:33)
      at Object.<anonymous> (../../tools/jest-env.js:8:17)

```

Reply:
So we have essentially:
```
jest_test(
   config = "jest-config.js",
   data = [":jest-config-deps", "//packages/wrapper-lib:tests"]
)
```

Note that the dependency is:
```
jest-config.js => jest-env.js => @e2e/lib => @aspect-test/a
```
and the tests:
```
wrapper-lib:tests => wrapper-lib => @e2e/lib => @aspect-test/a
```

The first of those is failing: the `jest_test(config)` indirectly referencing `@aspect-test/a`. So how does adding the dep to this BUILD file of `wrapper-lib` fix that!?

Reply:
I'm assuming it is something to do with `jest-resolve` hacking how imports work. See `jest-resolve` in the stack trace...